pkg/apis/extensions/validation/validation.go

```diff
@@ -323,6 +323,20 @@ func validateIngressTLS(spec *extensions.IngressSpec, fldPath *field.Path) field
 	allErrs := field.ErrorList{}
 	// TODO: Perform a more thorough validation of spec.TLS.Hosts that takes
 	// the wildcard spec from RFC 6125 into account.
```

Comment:
Are we doing enough validation that we no longer need the todo? If not, can you modify the todo to point out exactly what's left, now that we're validating hostname